e2e/appActions.js

```diff
@@ -120,6 +120,29 @@ async function createDomainObjectWithDefaults(page, { type, name, parent = 'mine
     };
 }
 
+/**
+  * Creates a notebook object and adds an entry.
+  * @param {import('@playwright/test').Page} - page to load
+  * @param {number} [iterations = 1] - the number of entries to create
+  */
+async function createNotebookAndEntry(page, iterations = 1) {
```

Comment:
Since this function is a "compound" appAction, and is solely relating to notebook actions, I'm thinking we should keep this local to `tags.e2e.spec.js`. 

Let's move it back to `tags.e2e.spec.js`, and let's also extract this part into a `test.beforeEach()`:

```js
    //Go to baseURL
    await page.goto('./', { waitUntil: 'networkidle' });
```